### ISSUE TITLE
Bot does not accept invites from groups it was kicked from during its current lifecycle

### ISSUE THREAD

**Issue by mdingena** [alta-vr/js-tale#8]
## Context

When `js-tale` is running, it seems capable of accepting group invites, and also responds to being kicked by leaving the group. However, being re-invited to a group it was kicked from does not make `js-tale` accept the invite during the same "session". To accept an invite to a group twice, you have to restart the bot. It will then accept the pending invite upon booting.

## Desired outcome

The bot should be able to accept invitations and handle being kicked multiple times for the same server group over the course of its lifecycle. It's likely that bots running `js-tale` will only realistically reboot when new bot logic is being deployed. This means that a bot might run for days or weeks at a time and it should therefore be able to handle being invited and kicked from a group more than once.

## Steps to reproduce:

- Make sure you kick you bot from the server group before you run the bot.
- Run the bot.
- Invite the bot account. The bot accepts the invite.
- Kick the bot account. The bot leaves the group.
- Invite the bot account. :warning: **Nothing happens.**
- Restart the bot. Pending invite is accepted now.

**Comment by Narmdo** [comment]
This appears to be an issue with events to remove from the invite list not being sent on the server.

As a temporary work around, the following should work. Make sure to run this once at the start, before any invites are accepted.

```js
groupManager.groups.on('create', group =>
{
    const id = group.info.id;
    const invites = groupManager.invites;
    const index = invites.items.findIndex(item => item.info.id == id);
    
    if (index >= 0)
    {
        invites.items.splice(index, 1);
    }
}
```

**Comment by mdingena** [comment]
Fixed in v1.4.0
